src/acceptance-test/java/org/zalando/nakadi/repository/db/SubscriptionDbRepositoryTest.java

```diff
@@ -91,88 +86,87 @@ public void whenGetSubscriptionByIdThenOk() throws Exception {
 
     @Test
     public void whenGetSubscriptionByKeyPropertiesThenOk() throws Exception {
-
+        final String owningApplication = TestUtils.randomUUID();
         // insert subscription into DB
         final Subscription subscription = RandomSubscriptionBuilder.builder()
-                .withOwningApplication("myapp")
+                .withOwningApplication(owningApplication)
                 .withEventTypes(ImmutableSet.of("my-et", "second-et"))
                 .withConsumerGroup("my-cg")
                 .build();
         insertSubscriptionToDB(subscription);
 
         // get subscription by key properties and compare to original
-        final Subscription gotSubscription = repository.getSubscription("myapp", ImmutableSet.of("second-et", "my-et"),
+        final Subscription gotSubscription = repository.getSubscription(owningApplication,
+                ImmutableSet.of("second-et", "my-et"),
                 "my-cg");
         assertThat("We found the needed subscription", gotSubscription, equalTo(subscription));
     }
 
     @Test
-    public void whenListSubscriptionsThenOk() throws ServiceUnavailableException {
```

Comment:
No, it is not replaced, cause other tests present there as well which are testing this kind of call